worm.py

```diff
@@ -17,7 +17,7 @@ class Genes(Enum):
 genes_variations = [Genes.HEALTH, Genes.DAMAGE, Genes.ENERGY, Genes.DEFENSE]
 
 
-def create_genome():
+def create_genome() -> List:
```

Comment:
It's better to specify a List of what is expected. If you don't know, make List[Any]. If you have problems with forward declaration List[Genes] you can make List['Genes'] with string name inside.